api/src/main/java/com/exadel/frs/core/trainservice/controller/FaceController.java

```diff
@@ -75,7 +75,7 @@ public FaceResponseDto addFaces(
             @RequestParam("subject")
             final String faceName,
             @ApiParam(value = "The minimal percent confidence that found face is actually a face.")
-            @RequestParam(value = "det_prob_threshold", required = false)
+            @RequestParam(value = "det_prob_threshold", required = false, defaultValue = "0")
```

Comment:
Why default value is 0?

Reply:
Means all values of threshold are suitable